### DISCUSSION TITLE
-----BEGIN AGE ENCRYPTED PAYLOAD----- instead of FILE ...

### DISCUSSION THREAD

**Discussion by Ch1ffr3punk** [FiloSottile/age#236] [General]
Hi,
while both, age and GnuPG are using armor headers/footers like MESSAGE or in age's case FILE
why not use a generic term like PAYLOAD? I am no native English speaker but something like
PAYLOAD, or maybe a word like CONTENT, would look IMHO better when sending messages and not only files.

Well, only a suggestion.

**Comment by str4d** [comment]
To me, this is working as intended. age is designed to be a file encryption tool, not a generic message encryption layer. The use of `FILE` in the PEM encoded format is a useful reminder in the "user interface" of the age format.

**Comment by FiloSottile** [comment]
Indeed, we don't want people to use age for messaging, because it would be a pretty lousy messaging encryption tool: no forward secrecy, no ratcheting, no authentication... age is optimized for file encryption, so the occasional reminder of that is something we want to keep, although I agree PAYLOAD would be a good generic word.

**Comment by Ch1ffr3punk** [comment]
Well, at least people have the option to use age as message encryption
tool, because the public key fits theoretically nicely in an SMS, which you
can't to with OpenPGP.

BTW. I already use age for messages with friends, on an offline computer.

Regards
Stefan
